File: api/organization_branding.go
package api

import (
	"github.com/google/uuid"
)

type UpsertOrganizationBrandingRequest struct {
	Title          *string    `json:"title"`
	Description    *string    `json:"description"`
	LogoImageID    *uuid.UUID `json:"logoImageId"`
	PageTitle      *string    `json:"pageTitle"`
	FaviconImageID *uuid.UUID `json:"faviconImageId"`
}

File: sdk/js/src/types/organization-branding.ts
import {BaseModel} from './base';

export interface OrganizationBranding extends BaseModel {
  title?: string;
  description?: string;
  logoImageId?: string;
  appDomain?: string;
  registryDomain?: string;
  emailFromAddress?: string;
  pageTitle?: string;
  faviconImageId?: string;
}
